mikeio1d/res1d.py

```diff
@@ -488,6 +488,11 @@ def start_time(self) -> datetime:
     def end_time(self) -> datetime:
         """End time of the result file."""
         return from_dotnet_datetime(self.data.EndTime)
+    
+    @property
+    def n_time_steps(self) -> int:
```

Comment:
I suggest to consider less cryptic naming: number_of_time_steps vs. n_time_steps :-).

Reply:
Good point, I'll change that